src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/BaseDefaultHeartbeatPropertyProvider.cs

```diff
@@ -4,11 +4,9 @@
     using System.Collections.Generic;
     using System.Linq;
     using System.Reflection;
+    using System.Runtime.InteropServices;
```

Comment:
why? is it needed?

Reply:
We are referencing RunTimeInformation within this file which applies to both .net 1.3 and .net 2.0. So I moved it out of #if. Please let me know if I'm misunderstanding something here